修复一些 README 错误 (#1)
Ext1nguisher/PCLhomepage: README.md

# PCLhomepage
***A simple PCL homepage for all!***
    
This mainpage has the following advantages that make you use it:

> * Easy to use:It simply has three cards!  It doesn't even need you to scroll!
> * Easy to change by yourself:We allow you to change this mainpage _BY YOUR OWN_! (Of course,not to sell it.)

Reviewer: ```suggestion
* Easy to use:It simply has three cards! It doesn't even need you to scroll!
* Easy to change by yourself: We allow you to change this mainpage _BY YOUR OWN_! (Of course, not to sell it.)
```

> * Easy to change by yourself:We allow you to change this mainpage _BY YOUR OWN_! (Of course,not to sell it.)

Reviewer 2: ```suggestion
* Easy to change by yourself:We allow you to change this mainpage _BY YOUR SELF_! (Of course,do not to sell it.)
```

> * Fantastic look:The colors in card seem to be _ICE FREEZING_ builtiful in all scenario!

Reviewer: ```suggestion
* Fantastic look: The colors in card seem to be _ICE FREEZING_ builtiful in all scenario!
```

> * Fantastic look:The colors in card seem to be _ICE FREEZING_ builtiful in all scenario!

Reviewer 2: ```suggestion
* Fantastic look:The colors in cards seem to be _ICE FREEZING_ builtiful in all scenario!
```

> ***Thank you for chosing us!***

Reviewer: ```suggestion
***Thank you for choosing us!***
```
